api/specs.proto

```diff
@@ -409,8 +409,15 @@ message SecretSpec {
 	// - golang: Go templating
 	Driver templating = 3;
 
-	// Driver is the the secret driver that is used to store the specified secret
+	// Driver is the the secret driver that is used to store the specified secret.  One example
```

Comment:
Updated to use `ExternalCA` - I guess we should support the same mechanism as the current root CA for external CAs - if we wanted to support plugins (such as the vault CA) that would be separate.